src/main/java/org/junit/runners/BlockJUnit4ClassRunner.java

```diff
@@ -328,14 +328,14 @@ protected Statement possiblyExpectingExceptions(FrameworkMethod method,
      * takes more than the specified number of milliseconds.
      */
     @Deprecated
-    protected Statement withPotentialTimeout(FrameworkMethod method,
-            Object test, Statement next) {
+    protected Statement withPotentialTimeout(FrameworkMethod method, Statement next) {
```

Comment:
Sorry, this change is likely due to poor merge conflict resolution on my part. I'll change this back. :wave: